custom_components/powercalc/data/signify/3216331P6/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "TP-Link HS110",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 5,
+        "VERSION": "master"
+    },
+    "name": "Philips Hue Aurelle (model 3216331P6, 46.5 W 220-240 V, 4200 lm)",
```

Comment:
Please remove the verbosity from the name, to be consistent with the other lights. This name is displayed in supported models listing.
Maybe "Hue White Ambiance Aurelle Panel 60x60"